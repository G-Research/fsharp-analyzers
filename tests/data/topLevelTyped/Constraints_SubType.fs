module CST

let v (c:'c) : unit =
    (c :> System.IDisposable).Dispose()
