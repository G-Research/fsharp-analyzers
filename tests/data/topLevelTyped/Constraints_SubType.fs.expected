GRA-UNTYPEDAPI-001 | Warning | (3,4 - 3,5) | `v` is not private, not fully typed and could be made private to file.
Missing constraint: 'c :> System.IDisposable
